design/ReconcileWithR.jl

```diff
@@ -335,7 +335,7 @@ Turing = "~0.29.3"
 PLUTO_MANIFEST_TOML_CONTENTS = """
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.9.3"
+julia_version = "1.9.1"
```

Comment:
are you sure you want to use a lower version? maybe even upgrade to 1.10 could be interesting?

Reply:
This was of course a change automatically generated by Pluto that reflects my Julia version is slightly older than the previous committer (I am guessing you, cos I have not changed my Julia install). I am just going to roll this change back. I like the idea of trying under 1.10 but not today!